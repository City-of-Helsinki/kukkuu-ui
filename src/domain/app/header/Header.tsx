import React from 'react';

import styles from './header.module.scss';
import Container from '../layout/Container';
import Navbar from './navbar/Navbar';

function Header() {
  return (
    <header className={styles.headerWrapper}>
      <Container>
        <Navbar />
      </Container>
    </header>
  );
}

export default Header;
