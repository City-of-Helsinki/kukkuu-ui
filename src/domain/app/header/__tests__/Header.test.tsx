import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from '../Header';

it('renders snapshot correctly', () => {
  const header = shallow(<Header />);
  expect(toJson(header)).toMatchSnapshot();
});
