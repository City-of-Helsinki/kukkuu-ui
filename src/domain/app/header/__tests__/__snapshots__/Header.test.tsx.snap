// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<header
  className="headerWrapper"
>
  <Container>
    <Navbar />
  </Container>
</header>
`;
